single-node-refactor/src/material_models/strength/user_defined_strength.h

```diff
@@ -65,12 +65,12 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace UserDefinedStrengthModel {
 
     KOKKOS_FUNCTION
-    static void calc_stress(const DCArrayKokkos<double>& elem_pres,
-        const DCArrayKokkos<double>& elem_stress,
-        const size_t mat_pt_lid,
+    static void calc_stress(const DCArrayKokkos<double>& MaterialPoints_pres,
+        const DCArrayKokkos<double>& MaterialPoints_stress,
+        const size_t MaterialPoints_lid,
```

Comment:
I think this value is for a single material point, so having the s at the end making it plural is a bit confusing.  I think the rest of the inputs are the data for all material points, so plural makes sense there.